.tx/config

```diff
@@ -0,0 +1,11 @@
+[main]
+host = https://www.transifex.com
+minimum_perc = 80
+lang_map = pt_BR: pt-rBR, ru_RU: ru-rRU, fr_FR: fr-rFR, es_MX: es-rMX, es_ES: es-rES, en_GB: en-rGB, ko_KR: ko-rKR
```

Comment:
Is it mandatory?

Reply:
The only thing that is not mandatory is "minimum_perc" can be lowest or highest number but need to be more than 1% at least

Reply:
My question is related to the `lang_map` because we add new translations often

Reply:
Yes we need to keep the lang_map we are checking (iOS and Android) if we can make this more flexible with a regular expression